apps/nextjs-website/src/components/StripeLink.tsx

```diff
@@ -17,35 +16,19 @@ const makeStripeLinkProps = ({
       icon,
     },
   }),
-  ...(buttonText && { buttonText }),
+  ...(buttonText &&
+    buttonText.length > 0 && {
+      ctaButtons: buttonText.map(({ icon, ...button }) => ({
+        ...button,
+        size: 'small',
+        ...(buttonText.icon && { endIcon: Icon(buttonText.icon) }),
```

Comment:
```suggestion
        ...(icon && { endIcon: Icon(icon) }),
```
`icon` has been destructured